code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -202,8 +202,8 @@
 		var/datum/job/previewJob
 		if(equip_preview_mob && SSjob)
 			// Determine what job is marked as 'High' priority, and dress them up as such.
-			if("Assistant" in job_low)
-				previewJob = SSjob.GetJob("Assistant")
+			if(ASSISTANT_TITLE in job_low)
+				previewJob = SSjob.GetJob(ASSISTANT_TITLE)
```

Comment:
Shouldn't we update preference version and add support code?

Reply:
Is it worth the effort? The only problem is that people who have assistant selected will lose their selection.